.travis.yml

```diff
@@ -57,5 +57,8 @@ before_install:
   - npm install -g npm@4
   - npm --version
 
+install:
+  - yarn install --no-lockfile --non-interactive
```

Comment:
This seems somewhat unrelated, I'd like to get CI passing without the runtime changes in ths PR so that we can evaluate things independently.

Also, it seems odd to use both `npm` and `yarn` in the same CI config. I would think that you should remove the various `npm` commands just above as well.

Reply:
yes, fair assessment about npm. Tbh I don't completely understand why this makes things work, but it all has to do with the dummy engines being linked into node_modules. Just installing directly with npm (or yarn) without the flags throws an error about node_modules missing inside the engine folders. There could be a different way to address this, but again I just followed the example config in ember-engines repo to get things working https://github.com/ember-engines/ember-engines/blob/master/.travis.yml#L54

Reply:
`npm` doesn't support `link:` as a prefix. if you want to use `npm` you have to use `file:` which does a symlink (but under `yarn` using `file:` means _duplicate copy_ into `node_modules` 🤕 ).

I'm super happy to move to `yarn`, I just want to be _fully_ in one camp or the other (and preferably in a separate "use yarn for CI" PR).

Reply:
Created a new PR for just the CI config changes: https://github.com/nickiaconis/ember-prefetch/pull/151